python/cudf/cudf/core/dataframe.py

```diff
@@ -2656,7 +2656,11 @@ def apply_rows(
 
         Parameters
         ----------
-        {params}
+        pessimistic_nulls : bool
+            Whether or not apply_rows should output nulls when an input is
+            null. If false, all outputs will be non-null, but will be the
+            result of applying func against the underlying column data, which
+            may be garbage.
```

Comment:
You wiped out the existing params with this. I think you need to put this below `{params}`

Reply:
done